bigtable/client/row_set.cc

```diff
@@ -31,7 +36,26 @@ RowSet RowSet::Intersect(bigtable::RowRange const& range) const {
       *result.row_set_.add_row_ranges() = std::get<1>(i).as_proto_move();
     }
   }
+  // Another special case: intersection is empty, return empty range
```

Comment:
Consider: `// Another special case: a RowSet() with no entries means "all rows", but we want "no rows"`